geomet/esri.py

```diff
@@ -0,0 +1,202 @@
+#  Copyright 2013 Lars Butler & individual contributors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import json
+#-------------------------------------------------------------------------
+def load(source_file):
+    """
+    Converts Esri Json File to GeoJSON.
+
+    :param source_file:
+        Path to a file that contains the Esri JSON data.
+    
+    :returns: 
+         A GeoJSON `dict` representing the geometry read from the file.
+
+    """
+    with open(source_file, 'r') as reader:
```

Comment:
the `load` functions operate on already-open file-like objects, not paths.

Reply:
Since we are working with JSON, I should be able to leverage the json.load method since it's not as complex like WKT/WKB.  I'll just use `json.load` to perform this operation